packages/components/organisms/f-registration/src/components/Registration.vue

```diff
@@ -470,7 +470,8 @@ export default {
                     registrationSource: 'Native',
                     marketingPreferences: []
                 };
-                await RegistrationServiceApi.createAccount(this.createAccountUrl, registrationData, this.tenant);
+
+                await RegistrationServiceApi.createAccount(this.$http, this.createAccountUrl, registrationData, this.tenant);
```

Comment:
See below seems annoying to have to pass `this.$http` to the service this way

Reply:
What would you recommend instead? If we performed our API calls in Vuex then we wouldn't need to; its because we are using JS files outside of the Vue scope.

Reply:
withought being able to use singleton can see this would not be easy to do, so for now might be best as you said to get it working